modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_8_0/UpgradeDDMStructure.java

```diff
@@ -79,23 +82,17 @@ protected void doUpgrade() throws Exception {
 		_upgradeDDMStructure();
 	}
 
-	private String _generateFieldSetName(DDMForm ddmForm) {
-		Map<String, DDMFormField> ddmFormFieldsMap =
-			ddmForm.getDDMFormFieldsMap(false);
-
-		String name = "FieldsGroup";
+	private String _generateDDMFormFieldName() {
+		String ddmFormFieldName = "Field";
```

Comment:
@marcosapmf, can we translate `Field` considering the form default locale? Do we have access to the language keys during this upgrade execution? If so, we should use it instead of the English translation.

Reply:
@natocesarrego, once fieldName is actually the field id, I don't think we should translate it according to the form default locale. Besides, it creates a fieldName similar to the ones created on [LPS-114619](https://issues.liferay.com/browse/LPS-114619).